Check/BestPractices/Services.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * @file
+ * Contains \SiteAudit\Check\BestPractices\Service.
+ */
+
+class SiteAuditCheckBestPracticesServices extends SiteAuditCheckAbstract {
+  /**
+   * Implements \SiteAudit\Check\Abstract\getLabel().
+   */
+  public function getLabel() {
+    return dt('sites/default/services.yml');
+  }
+
+  /**
+   * Implements \SiteAudit\Check\Abstract\getDescription().
+   */
+  public function getDescription() {
+    return dt('Check if the services file exists.');
+  }
+
+  /**
+   * Implements \SiteAudit\Check\Abstract\getResultFail().
+   */
+  public function getResultFail() {}
```

Comment:
Missing a message; `services.yml does not exist! Copy the default.service.yml to services.yml and see https://www.drupal.org/documentation/install/settings-file for details.`